src/main/java/io/swagger/inflector/processors/BinaryProcessor.java

```diff
@@ -55,7 +55,7 @@ public void enableType(MediaType type) {
 
     @Override
     public boolean supports(MediaType mediaType) {
-        return MediaType.APPLICATION_OCTET_STREAM_TYPE.equals(mediaType);
+        return SUPPORTED_TYPES.contains(mediaType);
```

Comment:
I think if we're going to remove this default behavior, we should add `MediaType.APPLICATION_OCTET_STREAM_TYPE` to the `SUPPORTED_TYPES` by default